From c11534351794d98b38896d28533961783f27225e Mon Sep 17 00:00:00 2001
From: WingMan Kwok <w-kwok2@ti.com>
Date: Fri, 15 Dec 2017 11:16:06 -0500
Subject: [PATCH 12/35] HACK: Documentation: dt: net: Add TI-PTP-BC bindings

Add DT binding information for TI's PTP boundary
clock internal monitor driver.

Signed-off-by: WingMan Kwok <w-kwok2@ti.com>
Signed-off-by: Lokesh Vutla <lokeshvutla@ti.com>
---
 .../devicetree/bindings/net/ti-ptp-bc.txt     | 33 +++++++++++++++++++
 1 file changed, 33 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/net/ti-ptp-bc.txt

diff --git a/Documentation/devicetree/bindings/net/ti-ptp-bc.txt b/Documentation/devicetree/bindings/net/ti-ptp-bc.txt
new file mode 100644
index 000000000000..064edf926300
--- /dev/null
+++ b/Documentation/devicetree/bindings/net/ti-ptp-bc.txt
@@ -0,0 +1,33 @@
+Texas Instruments PTP Boundary Clcok Internal Sync Monitor
+==========================================================
+
+Required properties:
+- compatible       : Should be
+                        "ti,am57-bc" for AM57xx SoCs
+Optional properties:
+- pps-sel0-gpios	: GPIO specifier, used to select PPS BC mux output
+- pps-sel1-gpios	: GPIO specifier, used to select PPS BC mux output
+
+Example:
+========
+
+In am57xx-idk-common.dts
+	ptp_bc: ptp_bc {
+		compatible = "ti,am57-bc";
+		status = "disabled";
+	};
+
+To enable for am572x-idk, in am572x-idk.dts:
+
+&ptp_bc {
+	status = "okay";
+};
+
+To enable for am571x-idk-pps, in am572x-idk-pps.dtso:
+
+&ptp_bc {
+	pps-sel0-gpios = <&gpio1 4 GPIO_ACTIVE_HIGH>;
+	pps-sel1-gpios = <&gpio6 7 GPIO_ACTIVE_HIGH>;
+
+	status = "okay";
+};
-- 
2.17.1

